Jenkinsfile

```diff
@@ -2,64 +2,17 @@
 
 REPOSITORY = 'release'
 
-node {
-  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
-
-  properties([
-    buildDiscarder(
-      logRotator(
-        numToKeepStr: '50')
-      ),
-    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
-  ])
-
-  try {
-    stage("Checkout") {
-      checkout scm
-    }
-
-    stage("Clean up workspace") {
-      govuk.cleanupGit()
-    }
-
-    stage("git merge") {
-      govuk.mergeMasterBranch()
-    }
-
-    stage("Configure Rails environment") {
-      govuk.setEnvar("RAILS_ENV", "test")
-    }
-
-    stage("bundle install") {
-      govuk.bundleApp()
-    }
-
-    stage("rubylinter") {
-      govuk.rubyLinter("app test lib")
-    }
-
-    stage("Precompile assets") {
-      govuk.precompileAssets()
-    }
-
-    stage("Run tests") {
-      govuk.runRakeTask("ci:setup:rspec default")
-    }
-
-    stage("Push release tag") {
-      govuk.pushTag(REPOSITORY, env.BRANCH_NAME, "release_${env.BUILD_NUMBER}")
+def releaseTests() {
+  stage("Run tests") {
+    echo "Running ci:setup:rspec default task"
+    withStatsdTiming("rake") {
+      sh("bundle exec rake ci:setup:rspec default")
```

Comment:
Not sure if `ci:setup:rspec` is actually needed, in a couple of other apps just `rake default` worked as well.